### PR TITLE
Realms/Staking Docs

### PR DESCRIPTION
**Pull request by jthiller**

### CODE REVIEW

File: docs/vote-escrow/realms.mdx [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+Once a stake has been delegated, it can be undelegated 
+<figure className="screensnippet-wrapper">
+    <img src={useBaseUrl('/img/realms/delegate.png')} style={{ maxHeight: 300 }} />
+    <figcaption>veHNT positions can be delegated the IoT or Mobile network.</figcaption>
+</figure>
+
+### Splitting a Stake
+A stake can be divided among multiple positions. This can be helpful if you wish to delegate your veHNT position to multiple networks (IOT or MOBILE) or set different lockup times to different positions.
+
+Be warned, splitting a [Landrush](#landrush) position after the initial 10 day period will result in losing the multiplier.
```
Comment (bryzettler) [inline docs/vote-escrow/realms.mdx]: From the tokens being split/transfered off the position.

File: docs/vote-escrow/realms.mdx [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+    <img src={useBaseUrl('/img/realms/issuestake.png')} style={{ maxHeight: 300 }} />
+    <figcaption>After pressing, "Lock Tokens", a Solana transaction is issued to create the stake.</figcaption>
+</figure>
+
+## Managing Stakes
+With Realms, it's a sizzling experience to take control of your HNT stakes, ensuring they're grilled to perfection.
+Chew through stake delegation, transfers, and cooldowns like a master chef. Just remember, in this well-done world, it's all about finding the perfect balance to steak your claim. <span className="🥩">🥩</span>
+
+### Delegating Stakes
+A stake along will allow its holder to participate in Helium Network (HNT) governance, however it will not earn any token return until it is delegated to a subDAO.
```
Comment (bryzettler) [inline docs/vote-escrow/realms.mdx]: A stake along?

File: docs/vote-escrow/realms.mdx [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+You lock 10.000 tokens with a lockup duration of one year. After two years you decide to start the unlocking process. Another year after that, you can withdraw the tokens.
+
+:::
+
+
+### Duration
+The duration of your cooldown is defined in days. The longer your tokens are staked, the greater the more veHNT will be issued. If you chose the 'Constant' lockup period, this duration takes affect after the 'Unlock' action is executed.
+
+While it is possible to set a higher cooldown than 5 years by using the 'Custom Duration', no additional vote weight multiplier will be added.
```
Comment (bryzettler) [inline docs/vote-escrow/realms.mdx]: The max lockup actually tops out at 4 years. The 5 years is a carry over from existing realms UI. Maybe we should just nix that?

File: docs/vote-escrow/realms.mdx [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+
+<img className="docsheader" src={useBaseUrl('/img/blockchain/realmsheader.png')} />
+
+Like many other Solana Ecosystem projects, Helium utilizes [Realms](https://realms.today) to organize and manage vote escrow tokens, delegation, and more.
+Stakes can be created and later managed within Realms at any time.
+
+## Landrush
+For the first 10 days following the Solana migration, HNT holders are elegible to stake their HNT for a 'landrush' bonus of 3x the lock amount.<sup>(</sup>[^1]<sup>,</sup>[^2]<sup>)</sup>
+
+If a Landrush stake is moved after the first 10 days have lapsed, the landrush bonus is forfeited.
```
Comment (bryzettler) [inline docs/vote-escrow/realms.mdx]: If you transfer out of or split tokens out of a Landrush position the bonus is forfeited.

File: docs/vote-escrow/realms.mdx [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+    <figcaption>veHNT positions can be delegated the IoT or Mobile network.</figcaption>
+</figure>
+
+### Splitting a Stake
+A stake can be divided among multiple positions. This can be helpful if you wish to delegate your veHNT position to multiple networks (IOT or MOBILE) or set different lockup times to different positions.
+
+Be warned, splitting a [Landrush](#landrush) position after the initial 10 day period will result in losing the multiplier.
+
+### Transfer a Stake
+Stakes can be transfered from one position into another within the same account. Positions can only be transfered to positions with a greater or equal duration. Only undelegated positions can be transfered. Staked tokens cannot be moved to other accounts.
```
Comment (bryzettler) [inline docs/vote-escrow/realms.mdx]: Unable to transfer to Landrush positions

File: docs/vote-escrow/realms.mdx (hunk 3 above)
Comment (jthiller) [reply docs/vote-escrow/realms.mdx]: Yea if we can update the realms UI for this it'd be great. Will up date the copy here.

**Review by ThornM9 — DISMISSED** [review]

**Review by abhay — APPROVED** [review]

**Pull request merged**